src/test/java/com/gui/AbstractTest.java

```diff
@@ -31,20 +31,21 @@ public abstract class AbstractTest {
 
     @BeforeMethod(alwaysRun = true)
     @Parameters(value = "browser")
-    public void setUpTest(String browser){
+    public void setUpTest(String browser) {
+        String hubURL = "http://localhost:4444/wd/hub";
```

Comment:
move the hardcoded String hubURL = "http://localhost:4444/wd/hub"; to a configuration file